packages/react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -858,7 +858,7 @@ export class SelectionZone extends React.Component<ISelectionZoneProps, ISelecti
 
     if (doc && element) {
       while (element && element !== doc.documentElement) {
-        if (isElementTabbable(element)) {
+        if (isElementTabbable(element) || element.hasAttribute('data-selection-index')) {
```

Comment:
What does this additional check add? I'm a bit wary of unintended changes because lots of things use `data-selection-index` but mostly it's not clear to me what the change does so hopefully my wariness will fade :)

Reply:
It's because we want the tags to be selectable, but not focusable (the focusable element is the clear button). In theory, IMO SelectionZone should not be paying attention to tab-ability at all (e.g. without this addition, a listbox that uses aria-activedescendant couldn't use SelectionZone to manage selection), and this whole check is super weird.

Fun bit of weirdness: since the `if` block is checking all elements between the click target and document, putting a picker inside a DetailsList cell, or inside a Dialog with `tabindex="0"` (which makes sense in some cases, e.g. v9) will randomly make it work.